src/diffusers/pipelines/deepfloyd_if/pipeline_if_img2img_superresolution.py

```diff
@@ -759,7 +759,7 @@ def preprocess_image(self, image: PIL.Image.Image, num_images_per_prompt, device
             image = [image]
 
         if isinstance(image[0], PIL.Image.Image):
-            image = [np.array(i).astype(np.float32) / 255.0 for i in image]
+            image = [np.array(i).astype(np.float32) / 175.0 - 1.0 for i in image]
```

Comment:
Where does the 175 come from? Bit of a weird number as 255 / 2 is more like ~128

Reply:
I shouldn't do math in my head when sleep deprived

Reply:
fixed!

Reply:
All good, that's why we have the review system ;-)